internal/app/repository/user_repository_integration_test.go

```diff
@@ -0,0 +1,114 @@
+package repository_test
+
+import (
+	"log"
+	"testing"
+
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/model"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/repository"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/user-api/constant"
+	"github.com/stretchr/testify/assert"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+func testCleanUp(db *gorm.DB) {
```

Comment:
I think the best way to clean a db between tests would honestly just be to create and drop one per test. Sharing one and relying on it to clean up after could cause a race condition as these tests are running and dropping.